# obstacles
Header header
int32 command_kind
int32 robot_id
